Controllers/ExtensionMappers/ExtensionMappers.cs

```diff
@@ -0,0 +1,43 @@
+﻿using EcommerceStore.Controllers.InputModels;
+using EcommerceStore.Controllers.ViewModels;
+using EcommerceStore.Data.Entities;
+using System;
+
+namespace EcommerceStore.Controllers.ExtensionMappers
+{
+    public static class ExtensionMappers
+    {
+        public static Brand MapToBrand(this BrandInputModel inputModel)
+        {
+            if (inputModel != null)
+            {
+                var brand = new Brand
+                {
+                    Name = inputModel.Name,
+                    FoundationYear = inputModel.FoundationYear
+                };
+
+                return brand;
+            }
+
+            throw new NullReferenceException("Brand input model was null");
+        }
+
+        public static BrandViewModel MapToBrandView(this Brand brand)
+        {
```

Comment:
Adding Map methods can be useful if these methods are reused in multiple places
In this case it's ok to do mapping inside select operator